Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4572,6 +4573,26 @@ def com_google_fonts_check_cjk_vertical_metrics(ttFont):
         yield PASS, 'Vertical metrics are good'
 
 
+@check(
+    id = 'com.google.fonts/check/cjk_sparse_glyphs',
+    conditions = ['is_cjk_font'],
+    description = """Hangul has 40 characters and it's the smallest CJK writing system. If a font contains less CJK glyphs than this writing system, raise a warn and inform the user that some glyphs may be encoded incorrectly."""
+)
+def com_google_fonts_check_cjk_sparse_glyphs(ttFont):
```

Comment:
here we need a single-line doc-string that works as the brief description for the check (as opposed to the much longer rationale text provided earlier)